src/search/heuristic.h

```diff
@@ -88,6 +88,8 @@ class Heuristic : public Evaluator {
     static void add_options_to_feature(plugins::Feature &feature, const std::string &name);
     static void add_options_to_feature(plugins::Feature &feature); // TODO 1082 remove this, just keep the one above
 
+    static std::tuple<std::shared_ptr<AbstractTask>, bool, std::string, utils::Verbosity> get_heuristic_parameters_from_options(const plugins::Options &opts);
```

Comment:
Do we generally use static methods for this kind of thing? Normally the convention we follow is to avoid access to private members where not needed, which means this could be a module-level function rather than being part of the class. This is in general the way modern C++ design leans.